### PR TITLE
Remote monitoring MD changes.

### PR DESCRIPTION
**Pull request by msvinaykumar**
Signed-off-by: msvinaykumar <vinakuma@redhat.com>

### CODE REVIEW

File: design/MonitoringModeAPI.md [hunk 1]
```diff
@@ -83,25 +83,27 @@ see [Update results](/design/UpdateResults.md)
...
            }
          }
        },
        "cpuUsage": {
          "results": {
            "general_info": {
+             "min": 0.14,
              "max": 0.84,
-             "mean": 0.12,
+             "avg": 0.12,
```
Comment (dinogun) [inline design/MonitoringModeAPI.md:96]: Need to add `sum` for cpuUsage as well

File: design/MonitoringModeAPI.md [hunk 2]
```diff
@@ -110,34 +112,39 @@ see [Update results](/design/UpdateResults.md)
...
-             "mean": 50.21,
+             "avg": 50.21,
              "units": "MiB"
            }
          }
        },
        "memoryLimit": {
          "results": {
            "general_info": {
+             "min": 50.6,
```
Comment (dinogun) [inline design/MonitoringModeAPI.md:123]: Don't need `min` for `memoryLimit`

File: src/main/java/com/autotune/common/data/result/Aggregator.java [hunk 3]
```diff
@@ -46,29 +46,29 @@ public void setUnits(String units) {
...
 
-    public void setMax(double max) {
-        this.max = max;
+    public void setMin(Double min) {
+        this.min = min;
     }
 
     @Override
     public String toString() {
         return "GeneralInfoResult{" +
```
Comment (dinogun) [inline src/main/java/com/autotune/common/data/result/Aggregator.java:67]: This needs to be changed to `Aggregator` as well

File: src/main/java/com/autotune/common/data/result/Aggregator.java [hunk 4]
```diff
@@ -15,26 +15,26 @@
  *******************************************************************************/
 package com.autotune.common.data.result;
 
-public class GeneralInfoResult {
-    private double sum;
-    private double avg;
-    private double min;
-    private double max;
+public class Aggregator {
```
Comment (dinogun) [inline src/main/java/com/autotune/common/data/result/Aggregator.java:18]: Lets change this to `AggregationInfo` or `AggregationInfoResult`

**Comment by msvinaykumar** [comment]
Deleting this branch due to incorrect merge done by me.

**Pull request closed without merging**